rules/REQUEST-942-APPLICATION-ATTACK-SQLI.conf

```diff
@@ -583,10 +592,53 @@ SecRule ARGS_NAMES|ARGS|XML:/* "@rx (?i)[\s'\"`()]*?\b([\d\w]+)\b[\s'\"`()]*?(?:
     tag:'paranoia-level/2',\
     ver:'OWASP_CRS/3.4.0-dev',\
     severity:'CRITICAL',\
-    multiMatch,\
-    setvar:'tx.sql_injection_score=+%{tx.critical_anomaly_score}',\
-    setvar:'tx.anomaly_score_pl2=+%{tx.critical_anomaly_score}'"
+    setvar:'tx.942130_lhs=/%{TX.1}/',\
+    setvar:'tx.942130_matched_var_name=%{matched_var_name}',\
+    chain"
+    SecRule TX:942130_lhs "@within /%{TX.2}/" \
```

Comment:
Since we compare a single value now, we could just use `@eq` and remove the delimiters from the variable and the operator.

Reply:
Agreed.